hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -4527,4 +4527,12 @@
       maximum number of buckets across all volumes.
     </description>
   </property>
+  <property>
+    <name>ozone.om.rocksdb.manual.wal.flush</name>
+    <value>false</value>
+    <tag>OZONE, OM</tag>
```

Comment:
Is it ok to have different tags between `ozone.om.rocksdb.manual.wal.flush` and `rocksdb.manual_wal_flush`? (`OZONE, OM`, `OM, SCM, DATANODE`). Can `ozone.om.rocksdb.manual.wal.flush` also have tag `DATANODE`? Or `rocksdb.manual_wal_flush` doesn't have tag `DATANODE`?